scripts/fix_package_naming.py

```
@@ -0,0 +1,109 @@
+def get_canonical_name(name):
+    url = f"https://pypi.org/pypi/{name.lower()}/json"
+    try:
+        resp = requests.get(url, timeout=5)
+        if resp.ok:
+            pypi_name = resp.json()['info']['name']
+            return re.sub(r"[-_.]+", "-", name).lower() 
+    except Exception:
+        pass
+    return name 
+
+def normalize_pipfile_packages(pipfile_path):
```

Ruff (0.12.2)

Expected 2 blank lines, found 1

Add missing blank line(s)

(E302)